packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -181,6 +183,13 @@ export interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderI
   cellsByColumn?: {
     [columnKey: string]: React.ReactNode;
   };
+
+  /**
+   * Minimum width of the row.
```

Comment:
Can you add this to `IDetailsItemProps`, above, so that we can use it in other not-quite-a-row scenarios in the list?

Reply:
Resolved in https://github.com/OfficeDev/office-ui-fabric-react/pull/9565/commits/4e84ac47fbeb854228234fe9f0a6e321802909c6. Also had to update snapshots, API file, and disable a flaky test that I'll rework.

The test fires an extra time because we fixed the initial problem of the List not rendering in tests!